### PR TITLE
[SE-0326] Update status: Implemented

### PR DESCRIPTION
**Pull request by stevapple**

### CODE REVIEW

**Comment by DougGregor** [comment]
I don't believe this feature has been enabled by default everywhere for 5.6 yet. @xedin can comment and okay the merge when it is enabled.

**Comment by xedin** [comment]
Yeah, unfortunately the enabling had to be rolled back, I'm working through a couple of remaining issues and once re-landed, going to update this thread.

File: proposals/0326-extending-multi-statement-closure-inference.md [hunk 1]
```diff
@@ -3,10 +3,9 @@
 * Proposal: [SE-0326](0326-extending-multi-statement-closure-inference.md)
 * Author: [Pavel Yaskevich](https://github.com/xedin)
 * Review Manager: [Ben Cohen](https://github.com/airspeedswift)
-* Status: **Accepted (2021-11-15)**
-* Implementation: [apple/swift#38577](https://github.com/apple/swift/pull/38577), [apple/swift#39989](https://github.com/apple/swift/pull/39989)
+* Status: **Implemented (Swift 5.6)**
```
Comment (xedin) [inline proposals/0326-extending-multi-statement-closure-inference.md:6]: Please change 5.6 to 5.7

File: proposals/0326-extending-multi-statement-closure-inference.md [hunk 2]
```diff
@@ -3,10 +3,9 @@
...
+* Implementation: [apple/swift#38577](https://github.com/apple/swift/pull/38577), [apple/swift#40397](https://github.com/apple/swift/pull/40397)
```
Comment (xedin) [inline proposals/0326-extending-multi-statement-closure-inference.md:7]: Please add https://github.com/apple/swift/pull/41730 to the list.

**Review by xedin — APPROVED** [review]
Thank you!

**Comment by xedin** [comment]
@stevapple Would you like to write a CHANGELOG entry for this feature?

**Comment by stevapple** [comment]
@xedin Just opened apple/swift#41852 for this👌

**Pull request merged**